### PR TITLE
Add star=2.7.9a samtools=1.10 gawk=5.1.0 container

### PR DESCRIPTION
**Pull request by praveenraj2018**
A combination of star=2.7.9a samtools=1.10 gawk=5.1.0 container

### CODE REVIEW

**Comment by drpatelh** [comment]
May be worth bumping to the latest version of `samtools` too here @praveenraj2018 ?

File: combinations/hash.tsv [hunk 1]
```diff
@@ -141,3 +141,4 @@ asciigenome=1.16.0,bedtools=2.30.0
 r-recetox-aplcms=0.9.3,r-base=4.1.0,r-arrow=4.0.1	quay.io/bioconda/base-glibc-debian-bash:latest	0
 python=3.9,pandas=1.3.0,seaborn=0.11.0
 bwa=0.7.17,samtools=1.9,bcftools=1.9	quay.io/bioconda/base-glibc-debian-bash:latest	0
+star=2.7.9a,samtools=1.10,gawk=5.1.0
```
Comment (drpatelh) [inline combinations/hash.tsv:144]:
```suggestion
star=2.7.9a,samtools=1.13,gawk=5.1.0
```

Comment (drpatelh) [reply combinations/hash.tsv:144]: Just means we won't need to create a new container in the future if a later version of samtools is needed :)

Comment (praveenraj2018) [reply combinations/hash.tsv]: Done!

**Comment by bgruening** [comment]
Merci!

**Comment by praveenraj2018** [comment]
> Merci!

Thanks!

**Pull request merged**